test/mocks/server/mocks.cc

```diff
@@ -19,8 +19,9 @@ namespace Envoy {
 namespace Server {
 
 MockOptions::MockOptions(const std::string& config_path)
-    : config_path_(config_path), admin_address_path_("") {
+    : config_path_(config_path), config_yaml_(""), admin_address_path_("") {
```

Comment:
nit: `config_yaml_("")` is redundant, you can remove (as well as `admin_address_path_("")`)

Reply:
Done.